### PR TITLE
better logic for plural_s()

### PR DESCRIPTION
**Pull request by Seris02**
## About The Pull Request

makes it so that plural_s() actually follows most english rules
which is good
even though the proc is never used with this it might actually become useful (i have plans)

## Why It's Good For The Game

code improvement

## Changelog

:cl:
code: better logic for plural_s()
/:cl:

### CODE REVIEW

**Comment by Wallemations** [comment]
Any clue if you could merge `\s` functionality into this too? Or would that not be possible based off how `\s` works?

**Comment by Seris02** [comment]
> Any clue if you could merge `\s` functionality into this too? Or would that not be possible based off how `\s` works?

give me an example of how '\s' is used because I don't know what that is

**Comment by Wallemations** [comment]
> > Any clue if you could merge `\s` functionality into this too? Or would that not be possible based off how `\s` works?
> 
> 
> 
> give me an example of how '\s' is used because I don't know what that is
> 
> 
> 
> edit: nevermind found it myself, and uh, no it wouldn't be possible, that's a byond text macro rather than a proc

Aight no worries, just wanted to see if that were at all possible.

**Comment by LemonInTheDark** [comment]
You could have stuck this in with the change pr you want to do, just btw (Unless it was for downstream in which case 🗞️)
Doing it like this is fine too mind, but don't feel like you can't refactor and add at one time

**Comment by Seris02** [comment]
they are related but I'm not certain if my change that I want to make will get accepted, so I decided to just submit this one anyway because, better code

**Comment by LemonInTheDark** [comment]
absolutely based

File: code/__HELPERS/pronouns.dm [hunk 1]
```diff
@@ -40,13 +40,23 @@
...
 			. = "es"
 		else
-			. = "s"
+			switch(copytext_char(pluralize, -1))
+				if("s")
+					. = "es"
+				if("x")
+					. = "es"
+				if("z")
+					. = "es"
```
Comment (Watermelon914) [inline code/__HELPERS/pronouns.dm:57]:
```suggestion
				if("s", "x", "z")
```

Comment (Seris02) [reply code/__HELPERS/pronouns.dm:57]: holy shit what, you can do that?

**Comment by Seris02** [comment]
@LemonInTheDark it went stale so this is just a ping to get a maintainer's attention

**Pull request merged**